### PR TITLE
Bvef files

### PR DESCRIPTION
**Pull request by jbschiratti**
I propose the following PR which aims at solving the problem raised in #4070

### CODE REVIEW

**Comment by agramfort** [comment]
looks clean !

can you add an entry to what's new page?

File: doc/whats_new.rst [hunk 1]
```diff
@@ -115,6 +115,8 @@ Changelog
 
     - Add plotting of head positions as a function of time in :func:`mne.viz.plot_head_positions` by `Eric Larson`_
 
+    - Add .bvef extension (BrainVision Electrodes File) to :func:`mne.channels.read_montage` by `Jean-Baptiste Schiratti`_
```
Comment (agramfort) [inline doc/whats_new.rst]: this should be in the new Changelog section at the top

Comment (jbschiratti) [inline doc/whats_new.rst]: I moved the line up ! Tell me if it is OK

**Comment by agramfort** [comment]
thx @jbschiratti 

congrats for the first PR !

**Comment by sappelhoff** [comment]
Hi @jbschiratti 

I am coming across this PR many years later, and I am wondering:

1. where does the `85` factor come from when defining `pos`?
2. how do you know that theta and phi are in degrees and they ought to be converted to radians?
3. Do you know which coordinate system BrainVision uses? Because after running this code, the coordinate system is the FIFF HEAD coordinate system ... so we must know something about converting from one to the other :thinking: 

for convenience, the relevant part of your code is pasted below:

        pol = np.deg2rad(np.array(theta))
        phi = [float(s.text) for s in root.findall("./Electrode/Phi")]
        az = np.deg2rad(np.array(phi))
        pos = _sph_to_cart(np.array([np.ones(len(az)) * 85., az, pol]).T)

I'd be very happy if you could take some time to explain :-)

See also previous attempts to solve this mystery: https://github.com/mne-tools/mne-python/pull/4531/files#r305528241

cc @jasmainak @drammock

**Comment by agramfort** [comment]
I doubt we can provide much details on this with @jbschiratti (was a post
doc in my lab)

Disclaimer we have never been users of brainvision / brain products
ourself. Just tried
to make it work for people.

So anything you find broken and can be fixed, as long as it can be tested
and documented,
is very welcome.

>

**Pull request merged**